RandLAPACK/drivers/rl_bqrrp.hh

```diff
@@ -428,7 +430,7 @@ int BQRRP<T, RNG>::call(
                 qr_tall_t_dur  += duration_cast<microseconds>(qr_tall_t_stop - qr_tall_t_start).count();
                 apply_transq_t_start = high_resolution_clock::now();
             }
-        } else if (this -> qr_tall == "cholqr") {
+        } else if (this -> qr_tall == BQRRP<T, RNG>::SubroutineType::qr_tall_subroutines::cholqr) {
```

Comment:
As before, you don't need the fully qualified name (prefix ``BQRRP<T, RNG>::`` shouldn't be needed).

Reply:
Same comment as before

Reply:
Fixed.